<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameFileManager</class>
 <widget class="QDialog" name="NewGameFileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="importStocksBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Import Stocks From File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numUsersBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter # of Users</string>
   </property>
  </widget>
  <widget class="QLabel" name="numUsersLbl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="daysPerTurnBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Days/Turn</string>
   </property>
  </widget>
  <widget class="QLabel" name="daysLbl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="totalDaysBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Total Days</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalDaysLbl">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>470</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
